patches/nginx-1.11.2-privilege_process.patch

```diff
@@ -0,0 +1,208 @@
+diff --git a/src/core/nginx.c b/src/core/nginx.c
```

Comment:
Hmm, seems like you forgot to apply this patch to the nginx core by editing `util/mirror-tarballs`.